openhtf/exe/triggers.py

```diff
@@ -63,9 +57,14 @@ def AutoStop(dummy_dut_id):  # pylint: disable=invalid-name
 def PromptForTestStart(message='Provide a DUT ID in order to start the test.',
                        text_input=True):
   """Make a test start trigger based on prompting the user for input."""
+  # Cache the result in case we're looping.
```

Comment:
Hm, 'looping' doesn't necessarily mean for the same DUT.  It's actually not really clear what 'looping' really means, and we've talked about removing it entirely and leaving it up to the caller, doing something like:
```
while True:
  my_test.Execute()
```
Is easy enough.  For now, I think we should make internal 'looping' as closely match the behavior you'd get with a while loop like that (so re-prompt for a new dut_id each time).